odata-url-conventions/4 Resource Path.md

```diff
@@ -959,4 +959,34 @@ Content-Type: text/plain
 
 $filter=[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]
 ```
+This POST request would result from submitting the HTML form
+```html
+<form method="post" action="http://host/service/People/$query"
+      enctype="text/plain">
+  <input name="$filter"
+    value='[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]'>
+</form>
+```
+:::
+
+::: example
+Example ##ex: passing multiple system query options in the request body
+```
+POST http://host/service/People/$query
+Content-Type: application/x-www-form-urlencoded
+
+$filter=[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]&
```

Comment:
I thought this content-type would encode space as `+`.

Reply:
Indeed, it produces `%24filter=[FirstName,LastName]+in+[["John","Doe"],...`. Does OData accept this? Or can we demand that it accepts this in the case of `/$query`?

Reply:
See https://issues.oasis-open.org/browse/ODATA-1540, where we want to entice servers to accept this (and discourage clients to use it 😄)